### DISCUSSION TITLE
How to get search working

### DISCUSSION THREAD

**Discussion by Topener** [docsifyjs/docsify#1592] [Q&A] [answered]
Hi all,

I'm currently setting up a new docs site and want to enable search. However, it doesn't matter what I do search seems to not be active at all, besides showing the search box.

I added the plugin, I added `"search": "auto"` to the config, the search-box shows up, but it doesn't respond to anything I type in. It doesn't show "no results" either. 

Am I missing something obvious here? 

I also tried more complex configuration, but no luck.

I have a custom sidebar, searchbox shows up there. Currently only have 3 extra `.md` pages. I also added some other plugins, but disabling those don't change anything either.

To clarify, I'm also not seeing any errors in the console. Search works on other docsify sites for same browser.

**Comment by sy-records** [comment, marked as answer]
You should provide the address of the site with the problem, or a sandbox where the problem can be reproduced

**Reply by Topener** [reply]
You're right, it's here: https://sharp-lovelace-ddcd32.netlify.app

**Reply by sy-records** [reply]
You need to remove a docsify.min.js

```diff
<script src="//cdn.jsdelivr.net/npm/docsify@4"></script>
<script src="https://cdn.jsdelivr.net/npm/docsify-tabs@1"></script>
<script src="https://unpkg.com/docsify-copy-code@2"></script>
<script src="https://cdn.jsdelivr.net/npm/docsify-progress@latest/dist/progress.min.js"></script>
<script src="//cdn.jsdelivr.net/npm/@alertbox/docsify-footer/dist/docsify-footer.min.js"></script>
-<script src="//cdn.jsdelivr.net/npm/docsify/lib/docsify.min.js"></script>
```

**Reply by Topener** [reply]
Thanks, removed it, but still same result.

**Reply by sy-records** [reply]
You also need to remove the docsify-progress.

```diff
<script src="//cdn.jsdelivr.net/npm/docsify@4"></script>
<script src="https://cdn.jsdelivr.net/npm/docsify-tabs@1"></script>
<script src="https://unpkg.com/docsify-copy-code@2"></script>
-<script src="https://cdn.jsdelivr.net/npm/docsify-progress@latest/dist/progress.min.js"></script>
<script src="//cdn.jsdelivr.net/npm/@alertbox/docsify-footer/dist/docsify-footer.min.js"></script>
```

![image](https://user-images.githubusercontent.com/33931153/122744131-49de7d80-d2ba-11eb-845e-cfa89ed6f9a3.png)

**Reply by Topener** [reply]
Oh wow that actually did the trick. I guess those two are incompatible then, a bummer, but no big deal. Thanks!
